app/controllers/contacts_controller.rb

```diff
@@ -38,9 +38,10 @@ class ContactsController < ApplicationController
   end
 
   def website_create
-    contact_recipient = params.require(:contactRecipient)
-    contact_params = params.require(contact_recipient)
-    requestor_details = current_user || params.require(:userData)
+    formValues = JSON.parse(params.require(:formValues)).deep_symbolize_keys
```

Comment:
```suggestion
    formValues = JSON.parse(params.require(:formValues), symbolize_names: true)
```

Reply:
Oh I didn't knew that can be done to "deep" symbolize keys for nested hash. Accepted suggestion, thanks.